provider-directory/src/main/resources/application.properties

```diff
@@ -3,3 +3,19 @@ server.ssl.key-store-type=JKS
 server.ssl.key-store=unset
 server.ssl.key-store-password=unset
 server.ssl.key-alias=unset
+
+capability.status=draft
+capability.id=health-api-argonaut-capability
```

Comment:
Possibly need to update to `health-api-provider-directory-capability`.

Reply:
done